README.md

```diff
@@ -1,96 +1,290 @@
+
+![DeepLab](https://lh3.googleusercontent.com/proxy/NBeYel6jJVYERa5NSQh1wiQQoSymzig7tvCiOi0v2Km4SlaBhwSN7ldNfC6DI8R8fRai9BqWDARxkr-wh9ZrKxA)
+
+## References
+
+[arxiv](https://arxiv.org/pdf/1606.00915.pdf), 2017.
+
+Liang-Chieh Chen, George Papandreou, Senior Member, IEEE, Iasonas Kokkinos, Member, IEEE, Kevin Murphy, and Alan L. Yuille, Fellow, IEEE
+
+[Github code](https://github.com/tensorflow/models/tree/master/research/deeplab)
+
+## GOOGLE EARTH ENGINE
+
+Earth Engine is a platform for scientific analysis and visualization of geospatial datasets, for academic, non-profit, business and government users.
+
+Earth Engine hosts satellite imagery and stores it in a public data archive that includes historical earth images going back more than forty years. The images, ingested on a daily basis, are then made available for global-scale data mining.
+
+Earth Engine also provides APIs and other tools to enable the analysis of large datasets.
+
+[Google Earth Engine](https://earthengine.google.com/)
+
+![Google Earth Engine](https://earthengine.google.com/static/images/annotated_playground.png)
+
+## GOOGLE MAPS API
+
+The Maps JavaScript API lets you customize maps with your own content and imagery for display on web pages and mobile devices. The Maps JavaScript API features four basic map types (roadmap, satellite, hybrid, and terrain) which you can modify using layers and styles, controls and events, and various services and libraries.
+
+[Google Maps Platform](https://cloud.google.com/maps-platform)
+
+![Google Map API](https://developers.google.com/maps/documentation/javascript/images/panoramaTiles.png)
+
+# INFRASTRUCTURE
+
+All Climate Change AI project infrastructure use components of **Google Cloud Platform**. The back end has been deployed on a GPU enabled **Compute Engine** instance.
+
+## GOOGLE CLOUD PLATFORM
+
+GCP consists of a set of physical assets, such as computers and hard disk drives, and virtual resources, such as virtual machines (VMs), that are contained in [Google’s data centers](https://www.google.com/about/datacenters/) around the globe. Each data center location is in a _region_. Regions are available in Asia, Australia, Europe, North America, and South America. Each region is a collection of _zones_, which are isolated from each other within the region. Each zone is identified by a name that combines a letter identifier with the name of the region. For example, zone `a` in the East Asia region is named `asia-east1-a`.
+
+This distribution of resources provides several benefits, including redundancy in case of failure and reduced latency by locating resources closer to clients. This distribution also introduces some rules about how resources can be used together.
+
+[Google Cloud Platorm](https://cloud.google.com/)
+
+![Google Cloud Platform](https://1.bp.blogspot.com/-iXDDeOlKnx8/UqEh-DEil7I/AAAAAAAAAVI/9Uz2tdIAYNI/s1600/Screen+Shot+2013-12-03+at+6.13.03+PM+%281%29.png)
+
+# Climate Change AI (CCAI) Backend
```

Comment:
Please don't just paste the original README at the end here. Integrate your new content into the existing content.